### PR TITLE
add indonesia translation

### PR DESCRIPTION
**Pull request by civiel024**
Revisi done

### CODE REVIEW

File: i18n/id/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- \usepackage{fancyhdr,ragged2e}
+- \lhead{\parbox[t]{0.5\textwidth}{\RaggedRight\rightmark\strut}}
+- \rhead{\parbox[t]{0.5\textwidth}{\RaggedLeft\leftmark\strut}}
+- \setlength{\headheight}{5\baselineskip}
+- \pagestyle{fancy}
+- \fancyfoot[LE,RO]{© 2022 tea.inc.}
+  - \fancyfoot[L]{v1.0.3+id}   # expand these variables!
+lang: id      # https://pandoc.org/MANUAL.html#language-variables
+dir: ltr      # language direction; ltr:left-to-right or rtl:right-to-left
+header-includes:
```
Comment (mfts) [inline i18n/id/metadata.yml:81]: please remove this. It's redundant and causes an error

File: i18n/id/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- id: web3
+  url: https://research.web3.foundation/en/latest/polkadot/overview/2-token-economics.html
+header-includes:
+- \usepackage{fancyhdr,ragged2e}
+- \lhead{\parbox[t]{0.5\textwidth}{\RaggedRight\rightmark\strut}}
+- \rhead{\parbox[t]{0.5\textwidth}{\RaggedLeft\leftmark\strut}}
+- \setlength{\headheight}{5\baselineskip}
+- \pagestyle{fancy}
+- \fancyfoot[LE,RO]{© 2022 tea.inc.}
+  - \fancyfoot[L]{v1.0.3+id}   # expand these variables!
```
Comment (mfts) [inline i18n/id/metadata.yml:78]: please indent this line properly

```suggestion
- \fancyfoot[L]{v1.0.3+id}   # expand these variables!
```

**Review by mfts — CHANGES_REQUESTED** [review]
There's currently an issue with compiling this to PDF. I will have to debug it.

File: i18n/id/metadata.yml (hunk 1 above)
Comment (mfts) [reply i18n/id/metadata.yml:81]: **only line 81**

File: i18n/id/white-paper.md [hunk 3]
```diff
@@ -0,0 +1,581 @@
...
+* itu meluruh secara eksponensial antara $\chi = \chi_{ideal}$ dan $\chi = 1$.
+
+Kami memilih penurunan eksponensial serupa untuk $\tau_{\textsc{all}}(\chi)$ karena ini menyiratkan penurunan eksponensial $\tau(\chi)$, dan kami ingin imbalan turun tajam melampaui $\chi_{ ideal}$ untuk mencegah satu paket menerima semua hadiah.
+
+Peluruhan didefinisikan sehingga tingkat inflasi berkurang paling banyak 50% ketika $\chi$ menggeser unit $d$ ke kanan $\chi_{ideal}$ – yaitu.
+$\tau_{\textsc{semua}}(\chi_{ideal} + d) \geq \tau_{\textsc{semua}} \cdot 0.5$.
+
+Kami mengusulkan fungsi tingkat bunga dan tingkat inflasi berikut, yang bergantung pada parameter $\chi_{ideal}$, $\tau_{ideal}$, $\tau_{0}$ dan $d$.
+
+\bemula{selaraskan*}
```
Comment (mfts) [inline i18n/id/white-paper.md:470]:
```suggestion
\begin{align*}
```

File: i18n/id/white-paper.md [hunk 4]
```diff
@@ -0,0 +1,581 @@
...
+&\tau_{\textsc{semua}}(\chi) = \tau_{0} + (\tau_{\textsc{semua}}(\chi_{ideal}) - \tau_{0})\frac{\chi }{\chi_{ideal}}\enspace\textrm{for}\;0 < \chi \leq \chi_{ideal} \\
+&\tau_{\textsc{semua}}(\chi) = \tau_{0} + (\tau_{\textsc{semua}}(\chi_{ideal}) - \tau_{0}) \cdot 2^{ (\chi_{ideal}-\chi)/d}\enspace\textrm{for}\;\chi_{ideal} < \chi \leq 1
+\end{selaras*}
```
Comment (mfts) [inline i18n/id/white-paper.md:473]:
```suggestion
\end{align*}
```

Comment (mfts) [inline i18n/id/white-paper.md:473]: Please check the formulas again. You have translated some of the subscript like _all_ and _ideal_ inside the formula that shouldn't be translated. 

I didn't check everywhere in the white-paper. But please make sure you don't translate formulas. 

Please refer to the locations in the original file here: https://github.com/teaxyz/white-paper/blob/main/white-paper.md?plain=1#L471 to make sure you are not overwriting any formula.

**Review by mfts — CHANGES_REQUESTED** [review]
The current build is breaking because you have translated some of the formula in the file. I mentioned inline the formula I noticed, but please check the entire document again.

File: i18n/id/metadata.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
+title: Protokol Terdesentralisasi untuk Remunerasi Open‑Source Ecosystem
```
Comment (mfts) [inline i18n/id/metadata.yml:1]:
```suggestion
title: Protokol Terdesentralisasi untuk Remunerasi Open Source Ecosystem
```

**Review by mfts — CHANGES_REQUESTED** [review]
Looks great, there's a minor issue in with the `-` character, so I suggested a change to a difference dash. 

Other than that it's ready to merge.

File: i18n/id/metadata.yml (hunk 5 above)
Comment (civiel024) [reply i18n/id/metadata.yml]: done sir

Comment (mfts) [reply i18n/id/metadata.yml]: Thank you

**Review by mfts — APPROVED** [review]
@Xercesblu3 please run the workflow and if it compiles successfully please merge :)

**Comment by mfts** [comment]
@Xercesblu3 all checks pass. Please merge 🚀

**Pull request merged**